### DISCUSSION TITLE
Build some additional hypnosis 'commands'

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#205] [Ideas]
- "Freeze"/"Unfreeze"
- "Follow Me"

Maybe added as custom activities?
Should only usable by triggering player (or all "allowed" members?)
Will need more LSCG communication between accounts

**Comment by littlesera** [comment]
created from #14

**Comment by dDeepLb** [comment]
freeze, unfreeze, follow me can be made as custom words too.
сustom activities is always good.
maybe setting to choose who can command you?
